src/doc_scan_service/session/retrieve/zoom.liveness.check.response.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const CheckResponse = require('./check.response');
+
+class ZoomLivenessCheckResponse extends CheckResponse {
+}
+
+module.exports = ZoomLivenessCheckResponse;
```

Comment:
I don't think we have this in Java or PHP (it's called `LivenessCheckResponse`).  Seeing as Java has already been released, do you think we should keep it the same?

Reply:
Renamed in 30a4f39